### PR TITLE
Rayn RTD Module: initial release

### PR DESCRIPTION
**Pull request by aleksandar-rayn**
## Type of change

- [x] Feature

## Description of change

Rayn Real-time Data Submodule

### CODE REVIEW

File: modules/raynRtdProvider.js [hunk 1]
```diff
@@ -0,0 +1,198 @@
...
+    } else if (window.raynJS && typeof window.raynJS.getSegtax === 'function') {
+      window.raynJS.getSegtax().then((segtaxData) => {
+        logMessage(LOG_PREFIX, `Segtax data from RaynJS: ${JSON.stringify(segtaxData)}`);
+        setSegmentsAsBidderOrtb2(reqBidsConfigObj, bidders, integrationConfig, segtaxData, checksum);
+        callback();
+      }).catch((error) => {
+        logError(LOG_PREFIX, error);
+        callback();
+      });
+    } else {
```
Comment (patmmccann) [inline modules/raynRtdProvider.js:181]: why not fall back to communicating with your endpoint from the module?

Comment (aleksandar-rayn) [reply modules/raynRtdProvider.js:181]: @patmmccann I'm not sure I understand what you mean exactly.
We have two steps, first we check for data in localStorage, then if our library is loaded we call it's function to retrieve the data and if none of the conditions are satisfied we just log a "no data" message and don't do anything.

Comment (patmmccann) [reply modules/raynRtdProvider.js:181]: I see that. How about a third step; if neither local storage nor the library exist, you communicate with an endpoint to gather the segments?

Comment (aleksandar-rayn) [reply modules/raynRtdProvider.js:181]: @patmmccann thank you for your observation. We may be adding this functionality at a later stage. For now we are relying on our customers to either run our library or populate local storage using our endpoints

**Review by patmmccann — APPROVED** [review]

File: modules/raynRtdProvider.js (hunk 1 above)
Comment (patmmccann) [reply modules/raynRtdProvider.js:181]: Please make sure your documentation reflects that

**Pull request merged**